src/gc/impl/conservative/gc.d

```diff
@@ -1673,8 +1654,12 @@ struct Gcx
 
     void* alloc(size_t size, ref size_t alloc_size, uint bits) nothrow
     {
-        return size <= 2048 ? smallAlloc(binTable[size], alloc_size, bits)
+        auto p = size <= 2048 ? smallAlloc(binTable[size], alloc_size, bits)
                             : bigAlloc(size, alloc_size, bits);
+
+        statsTracker.allocated(alloc_size);
```

Comment:
Since this function is performance critical, being about to turn this statement into a no-op is important.

Reply:
What do you mean, `pragma(inline)`?

Reply:
By no-op I mean no code is generated for the statement.

Reply:
I am afraid it is not possible - `GC.stats` is supposed to work for every program at any moment of time, it is not controlled by version or runtime flag.

Though considering how extremely expensive call to `alloc` is, I am not sure I understand concern about 2 extra integer additions either.